lib/pool.js

```diff
@@ -18,11 +18,11 @@ function parseUrl(url) {
 
 function Pool(url, options) {
   options = options || {}
-  var foo = parseUrl(url)
-  var protocol = require(foo.protocol)
+  var parsedUrl = parseUrl(url)
```

Comment:
I took the liberty of fixing this variable name, although it is not directly related to my change. Not fixing it felt wrong.